### ISSUE TITLE
Sending Pictures on slow connections

### ISSUE THREAD

**Issue by bokf** [deltachat/deltachat-android-ii#190]
I have experienced a similar behaviour to my android issue 393 https://github.com/deltachat/deltachat-android/issues/393
On a slow connection an image with 390KB was sent three times. This is better then before, where the image was sent more then 10 times. I used version v0.90.0.

**Comment by bokf** [comment]
Same problem on fast connection and large files ...
![bildschirmfoto_2018-12-23_19-01-51](https://user-images.githubusercontent.com/25562831/50386406-ede50f00-06e5-11e9-8883-7828768d6ee8.png)

**Comment by r10s** [comment]
interesting. sending attempts are started in sequences, not parallel.
i tried different things, however, i could not reproduce this behavior yet. any additional information would be welcome therefore.

**Comment by bokf** [comment]
In the email header there are different lines with times: 
Date: Sun, 23 Dec 2018 18:27:40 +0100 
=> is always the same, and shown in the list of mails.

This one is different for each mail:
Received: from knopi.disroot.org ([127.0.0.1])
by localhost (disroot.org [127.0.0.1]) 
(amavisd-new, port 10024)with ESMTP id DVISctW0QOHL; 
Sun, 23 Dec 2018 18:27:55 +0100 (CET)

The different emails have following times:
Sun, 23 Dec 2018 19:00:39 +0100 (CET)
Sun, 23 Dec 2018 19:00:27 +0100 (CET)
Sun, 23 Dec 2018 18:54:11 +0100 (CET)
Sun, 23 Dec 2018 18:53:56 +0100 (CET)
Sun, 23 Dec 2018 18:53:42 +0100 (CET)
Sun, 23 Dec 2018 18:53:28 +0100 (CET)
Sun, 23 Dec 2018 18:37:34 +0100 (CET)
Sun, 23 Dec 2018 18:37:18 +0100 (CET)
Sun, 23 Dec 2018 18:35:33 +0100 (CET)
Sun, 23 Dec 2018 18:35:19 +0100 (CET)
Sun, 23 Dec 2018 18:29:57 +0100 (CET)
Sun, 23 Dec 2018 18:29:43 +0100 (CET)
Sun, 23 Dec 2018 18:29:25 +0100 (CET)
Sun, 23 Dec 2018 18:29:11 +0100 (CET)
Sun, 23 Dec 2018 18:28:24 +0100 (CET)
Sun, 23 Dec 2018 18:28:09 +0100 (CET)
Sun, 23 Dec 2018 18:27:55 +0100 (CET)

**Comment by angelo-fuchs** [comment]
were you moving (physically or network-wise) while this happened?

**Comment by bokf** [comment]
No, during the second example I was at home. It is not very probable that my phone switched from one wlan to another. Normally it stays in the first wlan I logged in.
